src/cli.ts

```diff
@@ -224,14 +240,17 @@ const runMeasureCalculation = async () => {
 };
 
 const main = async (options: NormalizedOptions) => {
-  validateInputs(program.opts());
+  validateInputs(options);
 
   if (options.privateKey) {
     options.privateKey = await resolveJWK(options.privateKey);
   }
 
   // execute "Step 1": bulk data export
   if (options.fhirUrl && options.group) {
+    if (options.patientBundles || options.measureBundle) {
+      options._type = options._type?.concat(',Patient');
```

Comment:
This seems to create a broken query if you only pass in the measure parameter

`node build/cli.js -g cms122-10-patients -m cms122.json`

This error will be returned, since its looking for a types of ",Patient" with this query

/Group/cms122-10-patients/$export?_type=%2CPatient

```
 "message": "The following resourceTypes are not supported for _type param for $export: .",
      "stack":
          Error: The following resourceTypes are not supported for _type param for $export: 
```

The code may just want to be 

options._type = options._type?.concat('Patient');

Reply:
I noticed this error as well - looks like the server throws this error when an empty string is passed in as the value for `_type` because it interprets the empty string as an "unsupported" resource. The default configuration file in this repository sets `_type` to be an empty string. I updated the default configuration file to default the value of `_type` to be null, and I also added handling here to only append the `Patient` resource type is the length of `_type` is greater than zero.